src/administration/components/AdminPanel.jsx

```diff
@@ -17,160 +12,227 @@ const electron = window.require('electron');
 const ipcRenderer = electron.ipcRenderer;
 
 class Admin extends Component {
-    state = {
-        volunteers: [],
-        inputFirstName: '',
-        inputLastName: '',
-        isSaveButtonEnabled: false
-    };
```

Comment:
This is still a valid way to declare component state. Different syntax, same result. When in doubt, try to adapt to the existing code.